src/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: min-content;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: var(--white);
+  border: 1px solid var(--grey-border);
+  border-radius: 5px;
+  padding: 16px;
+}
+
+.card__product-image {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 0;
+}
+
+.card__product-name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--black);
+  margin-top: 40px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--grey-text);
+  margin-top: 4px;
+}
+
+.card__reviews {
+  margin-top: 16px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__reviews-text {
+  margin-top: 2px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
```

Comment:
you used this styles several times. 
Use `DRY` principle (Don't Repeat Yourself)
You can create some class with this styles and add it to elements that need